### DISCUSSION TITLE
How to add custom transcluded notes related to date ?

### DISCUSSION THREAD

**Discussion by irvrhingo** [SilentVoid13/Templater#1312] [Help]
Hi, 
I'm really new to Obsidian and Templater. 
I'm trying to redo my overall schedules using Periodic Notes, Templater and Calendar (I mainly use Daily, Weekly and Yearly Notes)
I've already managed to do Hyperlinks in the daily to be able to take me to the corresponding week/month/year note, and if it doesn't exist, create it and place it in the appropriate folder. So that's quite neat. 

But there's one thing I'm trying to do and I'm not sure how to do it. 
I have a yearly note called "PROJECTS & IDEAS 2024" that lives in my vault. I do one each year
I'd like to be able to insert it in all of my templates, so that it can appear as a transcluded note (that I then put in a callout or under a header so that I can fold/unfold it). But I'd like it to be automated, so that for each year, it automatically recognizes which note to pull. 

And I don't really know how to go about it. 

If anyone can help me, it'd be great. 
I think if I understand that, it'd be really helpful to me, because I'd like to use this same method to automate other things (having a transcluded link to my yearly note in my weekly notes for exemple). 

Thanks in advance !

**Comment by Zachatoo** [comment]
You should be able to do something like this

```
![[PROJECTS & IDEAS <% tp.date.now("YYYY") %>]]
```

**Reply by irvrhingo** [reply]
Hi Zach, 
thanks for the response. That's what I thought, but it doesn't work. It gives me a reply "the file PROJECTS & IDEAS <% tp.date.now("YYYY") %> couldn't be found". 

So I'm lost

**Reply by Zachatoo** [reply]
That means Templater didn't execute and replace `<% tp.date.now("YYYY") %>` with 2024.

There are many ways to execute Templater.

- You can do it manually with one of the Templater commands in the command palette (<kbd>CMD</kbd><kbd>P</kbd> by default). There's several commands in there depending on what you're trying to do.
- Turning on "Trigger Templater on new file creation" and creating a file that has Templater tags in it, for example with a template for the Daily Notes plugin or Periodic Notes plugin.
- Turning on "Trigger Templater on new file creation" and setting up folder templates, then creating a new file in the specified folder(s).

**Reply by Zachatoo** [reply]
Side note: if you get that error in your template file, that's normal. It shouldn't resolve the link correctly until Templater executes.

**Reply by irvrhingo** [reply]
ah yes, of course ! That was the issue ! alright, thanks so much Zach.
